Makefile.omdev.mingw

```diff
@@ -1,3 +1,17 @@
+# see if we have the new omdev
+PREVIOUS_GCC_VERSION = "5.3.0"
+GCC_VERSION := "`gcc -dumpversion`"
+OLD_OMDEV := $(shell expr "$(GCC_VERSION)" "==" "$(PREVIOUS_GCC_VERSION)")
+
+ifeq "$(OLD_OMDEV)" "1"
```

Comment:
Let's not support old and new OMDev at the same time. This is confusing and will make everything more complicated.
I would prefer to focus on getting the new version up and running.

Reply:
I think for now is wise to support both as we might need to switch back. I haven't run any tests with it yet, it works to build and run code from OMEdit but it needs proper testing.